### ISSUE TITLE
Page Metadata Not working

### ISSUE THREAD

**Issue by kumaraguru1735** [reflex-dev/reflex#2464]
**Describe the bug**
According to docs in https://reflex.dev/docs/pages/metadata/
```
meta = [
    {"name": "theme_color", "content": "#FFFFFF"},
    {"char_set": "UTF-8"},
    {"property": "og:url", "content": "url"},
]

@rx.page(
    title="My Beautiful App",
    description="A beautiful app built with Reflex",
    image="/splash.png",
    meta=meta,
)
def index():
    return rx.text("A Beautiful App")
   ```
but it not working

**To Reproduce**
 It seems like Meta looking str instead of list

**Specifics (please complete the following information):**
 - Python Version: 3.9.13
 - Reflex Version: 0.3.9
 - OS: windows 10
 - Browser (Optional): chrome
 
**Additional context**

**Comment by kumaraguru1735** [comment]
I seen the original code taking str instant of list for meta variable
```
def page(
    route: str | None = None,
    title: str | None = None,
    image: str | None = None,
    description: str | None = None,
    meta: str | None = None,
    script_tags: list[Any] | None = None,
    on_load: Any | list[Any] | None = None,
):
```

**Comment by Yummy-Yums** [comment]
working on this

**Comment by ElijahAhianyo** [comment]
> working on this @ElijahAhianyo

yeah sure, go for it

**Comment by kumaraguru1735** [comment]
fixed on #2477
